pkg/crds/yaml/generated/management.cattle.io_oidcclients.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.17.1
+  name: oidcclients.management.cattle.io
+spec:
+  group: management.cattle.io
+  names:
+    kind: OIDCClient
+    listKind: OIDCClientList
+    plural: oidcclients
+    singular: oidcclient
+  scope: Cluster
+  versions:
+  - name: v3
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: |-
+              APIVersion defines the versioned schema of this representation of an object.
+              Servers should convert recognized schemas to the latest internal value, and
+              may reject unrecognized values.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+            type: string
+          kind:
+            description: |-
+              Kind is a string value representing the REST resource this object represents.
+              Servers may infer this from the endpoint the client submits requests to.
+              Cannot be updated.
+              In CamelCase.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: Spec is the specification of the desired configuration for
+              the oidc client.
+            properties:
+              description:
+                type: string
+              redirectURIs:
+                items:
+                  type: string
+                type: array
+              refreshTokenExpirationSeconds:
+                description: |-
+                  A Duration represents the elapsed time between two instants
```

Comment:
This description does not fit the field name. It seems to be a generic description based on the field type.
This requires fixing in the oidc_provider_types.go

Reply:
fixed in https://github.com/rancher/rancher/pull/49558/commits/c6021f0ae0668f1aef5eaeeed4fd9ba9fdb9dbfa